kong/information.go

```diff
@@ -0,0 +1,53 @@
+package kong
+
+import (
+	"context"
+	"encoding/json"
+	"strings"
+)
+
+type Info struct {
+	Version       string `json:"version,omitempty" yaml:"version,omitempty"`
+	Configuration struct {
+		Database string `json:"database,omitempty" yaml:"database,omitempty"`
+		Portal   bool   `json:"portal,omitempty" yaml:"portal,omitempty"`
+		RBAC     string `json:"rbac,omitempty" yaml:"rbac,omitempty"`
+	} `json:"configuration,omitempty" yaml:"configuration,omitempty"`
+}
```

Comment:
This type definition needs to go to `kong/types.go` for consistency with other API data types.